src/components/common/map-difficulty.js

```diff
@@ -0,0 +1,20 @@
+export const mapDifficulty = (difficulty) => {
+  switch (difficulty) {
+    case 1.5:
+      return "medium";
```

Comment:
As the game level values: `easy`, `medium`, and `hard` are constant throughout the game and get used at multiple places. declare them as constants in a separate file or at the top of the current file. 
For any constant that you know beforehand(prior to program execution), use uppercase naming conventions.
Refer https://javascript.info/variables#uppercase-constants